### PR TITLE
Add page about the report misbehavior plugin - Closes #908

### LINKED ISSUE #908: Create page for the PoM plugin
Parent issue: #840

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #908

### CODE REVIEW

File: modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+		encryptedPassphrase: '', <3>
+		defaultPassword: '', <4>
+		fee: 100000000 <5>
+	},
+  }
+}
+----
+
+<1> `dataPath`: The data path for storing misbehavior related information captured from application.
+<2> `cleanupFrequency`(optional): Frequency of cleaning up the outdated data in second.
```
Comment (chris529) [inline modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc]:
```suggestion
<2> `cleanupFrequency`(optional): Frequency of cleaning up the outdated data in seconds.
```

File: modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<3> `encryptedPassphrase`(optional): The encrypted passphrase of the account that will report the misbehavior.
+<4> `defaultPassword`(optional): Default password to decrypt the encrypted passphrase.
+It is not recommended to set a default password in production environments for security reasons.
+<5> `fee`(optional): The fee required to report misbehavior transaction.
```
Comment (chris529) [inline modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc]:
```suggestion
<5> `fee`(optional): The fee required to report a misbehavior transaction.
```

**Review by chris529 — COMMENTED** [review]
Looks good, just a couple of very minor points 👍

File: modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+		fee: 100000000 <5>
+	},
+  }
+}
+----
+
+<1> `dataPath`: The data path for storing misbehavior related information captured from application.
+<2> `cleanupFrequency`(optional): Frequency of cleaning up the outdated data in second.
+<3> `encryptedPassphrase`(optional): The encrypted passphrase of the account that will report the misbehavior.
+<4> `defaultPassword`(optional): Default password to decrypt the encrypted passphrase.
```
Comment (shuse2) [inline modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc]: `defaultPassword` this seems not used right now, so let's remove it

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/references/lisk-framework/report-misbehavior-plugin.adoc]:
```suggestion
<5> `fee`(optional): The fee required to send a report misbehavior transaction.
```

**Review by shuse2 — DISMISSED** [review]

**Pull request merged**